Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -2741,6 +3023,7 @@ def acknowledge_warning_command(client: Client, change_path: str) -> Tuple[objec
         return f'{INTEGRATION_NAME} - Could not find any results for given query', {}, {}
 
 
+# Created by C.L.
```

Comment:
Please remove comment

Reply:
We have two individuals working on the same integration and only using the Xsoar's Web based development interface.
We are planning to get the CI/CD pipeline setup within this year. However, before that is ready, we are probably going to keep track of some of our work in this manner.